### ISSUE TITLE
QR codes not appearing in app

### ISSUE THREAD

**Issue by AJMiller** [jaredonline/google-authenticator#53]
When trying to generate a QR code, the google api returns a 502 response now. It appears that the child gem that creates the QR code (https://github.com/jhsu/google-qr) is using a deprecated API at Google. Switching to something that generates the QR codes internally may be a better long term solution, and more secure: https://github.com/whomwah/rqrcode

Thoughts?

**Comment by harisadam** [comment]
@AJMiller had the same issue today.

We use qr code in a json response in our current project so thats why url was such an easy way. To be compatible with our current code I created a `google_qr_to_base64` method that returns a base64 image an accepts a size parameter 
https://github.com/harisadam/google-authenticator/commit/312d619333832c636d7412f5370601e831bafc2e

**Comment by jaredonline** [comment]
I've merged @harisadam's PR and pushed up version `1.7.0` so you should be able to work around this now.

**Comment by AJMiller** [comment]
thanks @harisadam and @jaredonline. The rqrcode implementation looks simple enough. Might be worth exposing an instance of that qr code as a method so that other users can render it however they need to?
i.e.
```ruby
def qr_code_object
  RQRCode::QRCode.new(one_time_pass)
end

def one_time_pass
  ROTP::TOTP.new(
    google_secret_value,
    :issuer => google_issuer
  ).provisioning_uri(google_label).to_s
end
```
